sim/midas/src/main/scala/midas/platform/xilinx/AXI4Crossing.scala

```diff
@@ -170,6 +170,39 @@ class XilinxAXI4Bundle(val bundleParams: AXI4BundleParameters, val isAXI4Lite: B
       }
     }
   }
+
+  def tieoff(axiClock: Clock, axiReset: Bool): Unit = {
+    awid.foreach { _ := DontCare }
+    awaddr  := DontCare
+    awlen.foreach { _ := DontCare }
```

Comment:
Have your forays into Vivado synthesis issues warrant that we start tying things off to known values instead of `DontCare`, instead of things that may be implementation defined? I'm fine with this as is.

Reply:
Good to know `0.U` works, I prefer that.

Reply:
So long as valid is tied off you should be able to set the payload values to whatever you want, and then yeah, 0.U will have its width inferred accordingly.